src/dataDisplay/table.jsx

```diff
@@ -1,21 +1,522 @@
-'use strict';
-
-import React, { Component } from 'react';
 import ClassNames from 'classnames';
+import {
+    debounce,
+    get,
+} from 'lodash';
 import PropTypes from 'prop-types';
+import React from 'react';
+import ScrollBar from 'react-custom-scrollbars';
+import { withStyles } from '../styles';
+import Utils from '../utils/utils';
 import TableBody from './tableBody';
 import TableCell from './tableCell';
+import {
+    DEBOUNCE_WAIT_TIME,
+    TH_RESIZABLE_MAX_WIDTH,
+} from './tableConstants';
 import TableHeader from './tableHeader';
-import TableHeaderCell from './tableHeaderCell';
 import TableRow from './tableRow';
+import TableHeaderCell from './tableHeaderCell';
+
+const propTypes = {
+    /**
+     * Basic (default) styling.
+     */
+    basic: PropTypes.bool,
+    /**
+     * If `true`, all columns will be divided.
+     */
+    celled: PropTypes.bool,
+    /**
+     * The content of the Table.
+     */
+    children: PropTypes.node,
+    /**
+     * Override or extend the styles applied to Table.
+     */
+    classes: PropTypes.shape({
+        tableSticky: PropTypes.string,
+        tableStickyColumns: PropTypes.string,
+    }),
+    /**
+     * Add additional classes to the Table.
+     */
+    className: PropTypes.string,
+    /**
+     * If `true`, applies `width: auto` to the Table.
+     */
+    collapsing: PropTypes.bool,
+    /**
+     * If `true`, far left column divided from the rest of the columns.
+     */
+    definition: PropTypes.bool,
+    /**
+     * If `true`, columns can be evenly spaced.
+     */
+    fixed: PropTypes.bool,
+    /**
+     * Change the font size of typography in the Table.
+     */
+    fontSize: PropTypes.oneOf(Utils.sizeEnums()),
+    /**
+     * Deprecated prop.
+     */
+    fullWidth: PropTypes.bool,
+    /**
+     * Add an id to the Table.
+     */
+    id: PropTypes.string,
+    /**
+     * Table row and cells can be highlighted as if they have been selected.
+     */
+    selectable: PropTypes.bool,
+    /**
+     * Truncate copy within cells in the Table
+     */
+    singleLine: PropTypes.bool,
+    /**
+     * Change the cell's vertical size in the Table.
```

Comment:
"cell's" should be "cells'" (plural possessive instead of singular possessive)

Reply:
Fixed.

Reply:
Replaced with plain old plural, versus plural possessive, but all good (I think it was hard to see the ending apostrophe due to the ending double quote in "cells'" in my comment above).

It seems it's well past time to stop nit-picking and ship this sucka!  So, on that note ... resolving.